src/main/java/org/jenkinsci/test/acceptance/po/BuildWithParameters.java

```diff
@@ -49,6 +49,11 @@ public BuildWithParameters enter(List<Parameter> definitions, Map<String, ?> val
     }
 
     public void start() {
-        clickButton("Build");
+        if (Boolean.getBoolean("new-job-page.flag.defaultValue")) {
+            find(by.xpath("//dialog[@open]//div[@id='bottom-sticker']//button[contains(., 'Build')]"))
```

Comment:
`BuildWithParameters` is a dialog in the Job page now - so this whole class should probably be deprecated.  The Page `/build` is still present and directly accessible and uses the old format and anyone using this PO to access that page should still work.